core/embed/rust/src/micropython/macros.rs

```diff
@@ -164,3 +170,36 @@ macro_rules! obj_type {
         }
     }};
 }
+
+/// Construct an extmod definition.
+macro_rules! obj_module {
+    ($($key:expr => $val:expr),*) => ({
+        #[allow(unused_unsafe)]
+        unsafe {
+            use $crate::micropython::ffi;
+
+            static DICT: ffi::mp_obj_dict_t = ffi::mp_obj_dict_t {
+                base: ffi::mp_obj_base_t {
+                    /// SAFETY: FIXME extern static
+                    type_: unsafe { &ffi::mp_type_dict },
```

Comment:
@jpochyla can you advise if this is safe? Seems like a value that shouldn't be changing normally.

Reply:
Safe 👍

Reply:
Thanks, updated comment in d128f680d9d2dd267407647e608b38f501a6d737.